player/src/lib.rs

```diff
@@ -201,83 +201,25 @@ impl GlobalPlay for wgc::hub::Global<IdentityPassThroughFactory> {
                         .unwrap();
                 }
             }
-            A::CreateBindGroupLayout {
-                id,
-                ref label,
-                ref entries,
-            } => {
-                self.device_create_bind_group_layout::<B>(
-                    device,
-                    &wgt::BindGroupLayoutDescriptor {
-                        label: Some(label),
-                        entries,
-                    },
-                    id,
-                )
-                .unwrap();
+            A::CreateBindGroupLayout { id, desc } => {
```

Comment:
nit: naming these isn't useful any more, we should just have `A::CreateBindGroupLayout(id, desc)`